tracer/src/Datadog.Trace/Propagators/DatadogContextPropagator.cs

```diff
@@ -67,7 +72,6 @@ private DatadogContextPropagator()
             var samplingPriority = ParseUtility.ParseInt32(carrier, carrierGetter, HttpHeaderNames.SamplingPriority);
             var origin = ParseUtility.ParseString(carrier, carrierGetter, HttpHeaderNames.Origin);
             var propagatedTraceTags = ParseUtility.ParseString(carrier, carrierGetter, HttpHeaderNames.PropagatedTags);
-
```

Comment:
FYI sometimes we add whitespace like this intentionally to visually separate logical units of work. In this case, `parentId`, `samplingPriority`, `origin`, and `propagatedTraceTags ` are extracted from http headers using `ParseUtility`, and `traceTags` is handled differently.

I also separate block of code with whitespace, like in line 45 above:
```
var propagatedTagsHeader = context.PrepareTagsHeaderForPropagation();
<intentional space>
if (!string.IsNullOrEmpty(propagatedTagsHeader))
{
...
}
```

I general, I recommend leaving empty lines as-is so we don't start Formatting Wars and to avoid unnecessary noise in git diffs.

Reply:
Changed in 18d1179